kategory/src/main/kotlin/kategory/data/ListKW.kt

```diff
@@ -0,0 +1,49 @@
+package kategory
+
+typealias ListKindW<A> = HK<ListKW.F, A>
+
+fun <A> ListKindW<A>.ev(): ListKW<A> =
+        this as ListKW<A>
+
+class ListKW<A> private constructor(private val list: List<A>) : ListKindW<A>, Collection<A> by list {
+
+    class F private constructor()
+
+    fun <B> map(f: (A) -> B): ListKW<B> = ListKW(list.map(f))
+
+    fun <B> flatMap(f: (A) -> ListKW<B>): ListKW<B> = ListKW(list.flatMap { f(it).list })
+
+    operator fun plus(list: List<@UnsafeVariance A>): ListKW<A> = ListKW(this.list + list)
```

Comment:
That was in case the unsafe variance was related in any way to encoding Contravariant positions but this is not the case. If you want to see an example of that `Option#getOrElse`